egs/sprakbanken/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+# This is the original TDNN script before we introduced xconfigs.
+# See run_tdnn_1b.sh for comparative results.
+
+
+# by default, with cleanup:
+# local/chain/run_tdnn.sh
+
+# without cleanup:
+# local/chain/run_tdnn.sh  --train-set train --gmm tri3 --nnet3-affix "" &
+
+# note, if you have already run the corresponding non-chain nnet3 system
+# (local/nnet3/run_tdnn.sh), you may want to run with --stage 14.
+
+set -e -o pipefail
```

Comment:
The current situation is many scripts do have the 'set -e', and many do not.  Officially, either is OK right now.  It's not an ideal situation but that's the way it is.